

g++ programtest23.cpp -O3 -lpqxx -lpq  && ./a.out

terminate called after throwing an instance of 'std::bad_alloc'
  what():  std::bad_alloc
Aborted (core dumped)

write.csv2(df, "C:/Users/szczkr/Downloads/PZU.csv",na= '', row.names = FALSE, fileEncoding = "UTF-8",quote = FALSE)
